src/App.vue

```diff
@@ -23,6 +23,56 @@ export default {
     title: 'Welcome',
     // all titles will be injected into this template
     titleTemplate: '%s | aec.works',
+    htmlAttrs: {
+      lang: 'en',
+      amp: false,
+    },
+    meta: [
+      {
+        property: 'og:type',
+        content: 'website',
+      },
+      {
+        property: 'og:url',
+        content: 'https://aec.works',
+      },
+      {
+        property: 'og:title',
+        vmid: 'og:title',
+        template: (chunk) => `${chunk} | aec.works`,
+      },
+      {
+        property: 'og:description',
+        content:
+          'An open-source AEC community platform that promotes inclusive and enriching conversations through content-sharing and discussions.',
+      },
+      {
+        property: 'og:image',
+        content: require('@/assets/images/meta.png'),
+      },
+      {
+        property: 'twitter:card',
+        content: 'summary_large_image',
+      },
+      {
+        property: 'twitter:url',
+        content: 'https://aec.works',
+      },
+      {
+        property: 'twitter:title',
+        vmid: 'twitter:title',
+        template: (chunk) => `${chunk} | aec.works`,
+      },
+      {
+        property: 'twitter:description',
+        content:
+          'An open-source AEC community platform that promotes inclusive and enriching conversations through content-sharing and discussions.',
```

Comment:
thinking of "stronger" title... 🤔 
`An Open Source Community Platform for the AEC Industry`
or `The Open Source AEC Community Platform`
Thoughts?

Reply:
I like “The Open Source...” sounds legit